src/components/stegvisning/Ufoere/Ufoere.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+
+import { Alert, BodyLong, Button, Heading } from '@navikt/ds-react'
+
+import { Card } from '@/components/common/Card'
+import { ReadMore } from '@/components/common/ReadMore'
+import { logger, wrapLogger } from '@/utils/logging'
+import { getFormatMessageValues } from '@/utils/translations'
+
+import styles from './Ufoere.module.scss'
+
+interface Props {
+  isLastStep: boolean
+  onCancel: () => void
+  onPrevious: () => void
+  onNext: () => void
+}
+
+export function Ufoere({ isLastStep, onCancel, onPrevious, onNext }: Props) {
+  const intl = useIntl()
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault()
+
+    logger('button klikk', {
+      tekst: 'Neste',
+    })
+    onNext()
+  }
+
+  // React.useEffect(() => {
+  //     logger('alert', {
+  //       tekst: 'Uføretrygd og AFP',
+  //     })
+  // }, [])
```

Comment:
Kan fjernes?

Reply:
Venter på avklaring om det skal logges noe for det steget